docs/spring-boot-runtime/master.adoc

```diff
@@ -240,6 +240,8 @@ include::topics/proc_running-the-secured-booster-integration-tests-on-sb-vx.adoc
 include::topics/secured-mission-resources.adoc[leveloffset=+3]
 
 == Application Development
+The recommended approach for specifying and using supported and tested maven artifacts in a {SpringBoot} application is to use the OpenShift Application Runtimes {SpringBoot} BOM.
```

Comment:
Do we have a variable somewhere for the "OpenShift Application Runtimes" text? Not sure if it would fit into the present phrasing, but I guess care must be taken here.

Reply:
@Ladicek, we do have a `ProductName` variable in `document-attributes.adoc`, which resolves to `Red Hat OpenShift Application Runtimes` in the _downstream_ docs only.
In the upstream, it resolves to `Application Development on OpenShift` to avoid branding the upstream docs.
I don't think _Application Development on OpenShift_ would fit inn the current phrasing, so I'd be inclined to leave the paragraph as is for the time being. We can always define a new attribute later:)

Reply:
Sure, there's probably some reason why we don't use the phrase "Red Hat OpenShift Application Runtimes" in the upstream docs, and "OpenShift Application Runtimes" is suspiciously similar to that :-)